caffeine/core.py

```diff
@@ -47,12 +47,15 @@ def __init__(self, process_manager):
 
         self.__inhibitors = [
             GnomeInhibitor(),
-            XdgScreenSaverInhibitor(),
             XdgPowerManagmentInhibitor(),
             XssInhibitor(),
-            DpmsInhibitor(),
             XorgInhibitor(),
-            XautolockInhibitor()
+            XautolockInhibitor(),
+        ]
+
+        self.__screen_only_inhibitors = [
```

Comment:
I'd rather add an attribute to each one `inhibits_suspension`, and use that (instead of keeping two lists). More on this below.